src/sentry/tasks/performance_detection.py

```diff
@@ -29,7 +29,9 @@ def detect_performance_issue(data: Event):
         if rate and rate > random.random():
             # Add an experimental tag to be able to find these spans in production while developing. Should be removed later.
             sentry_sdk.set_tag("_did_analyze_performance_issue", "true")
-            with sentry_sdk.start_span(
+            with metrics.timer(
+                "performance.detect_performance_issue", sample_rate=0.01
```

Comment:
`sample_rate=0.01` at the rates we are currently collecting events, this should easily be stat. significant for percentiles, unlike the detection counts, exact numbers aren't really needed. Could even be lower, but this should be fine for now.